### ISSUE TITLE
Add input field on Scout Report for extra user info

### ISSUE THREAD

**Issue by mradamcox** [legiongis/hmsf-md#80]
A new field should be added to the Scout Report graph that collects information about state employee names. The reason for this is that parks and other entities will continue to share logins, typically one per park. This means that while the Scout ID field should contain the name of the user id, the new field can be used to add more specific information about the state user that is actually performing the condition assessment.

![image](https://user-images.githubusercontent.com/10427268/70877131-e3b8c200-1f81-11ea-96a2-b35aa4163f74.png)

**Update:**
Some new thoughts on this based on discussions with @kkemp85, and our expanding user base: "Your Scout ID" should be changed to "Your User ID", and the new node described in this ticket should be something like "Extra User Info" (maybe there's something better than "info" to use. This new field will hold more than just state employee names:
- Legacy scout ids that did not exactly match the actual scout username (or people who had accounts in the pre-Arches system, but not this one)
- Extra scouts who were along for the visit
- State employees as described above
- Any number of other types of alternative staff/user names, now that we will have more county admins, etc.

**Comment by mradamcox** [comment]
@coherit To test this out before doing it in production, I think the most important steps will be (locally): add this node (text) and name it (should be optional). Then make sure you can still access/edit existing scout reports (make sure you have some loaded first). Then export the resource model to the package, and reload the package with the test scout report data to make sure it all loads correctly. I really expect all of that to go off without a hitch but you never know.

Finally, when we're sure this really is as easy as we hope, we can make the change in the production GUI, and then export _that_ resource model from there to put in the fpan-data package.

Hope that makes sense.

**Comment by coherit** [comment]
@mradamcox I've created the new "State Employee Name" node as datatype=string and have it as part of the Step 2 branch as you mocked up. (Required me to export the branch and then append the new card.)

Unfortunately, while I can access previous scout reports and edit all other nodes, editing the new node does not give me the Add+ option (only Delete this record). I clicked the "re-index resources" button in the functions resource descriptors area, but that didn't change anything. 

I then ran python manage.py es index_database hoping that it was an elasticsearch issue. I got several of the following warnings, for archaeological sites:
`badly formed hexadecimal UUID string invalid nodegroupid participating in descriptor function.
`
Unfortunately, this didn't change anything. I am able to add new resources with the new node completed, but I cannot edit existing resources to add information to the new node. 

Is this expected behavior? I seem to remember in previous versions of Arches not being able to add new nodes midstream. 

Do you think this is going to require rebuilding the business data for the scout resource to set "State Employee Name": null for everything and importing the both the resource model and business data again? Let me know what you think -- I don't want to spend the time on that if you don't want to go down that path with production.

![image](https://user-images.githubusercontent.com/3217017/77190116-0ee60c80-6aaf-11ea-9910-5d05b137ad81.png)

**Comment by mradamcox** [comment]
Ok, to get as far as you did did you need to change the `OVERRIDE_RESOURCE_MODEL_LOCK` setting to True? If you didn't do that yet, maybe it will help with this editing issue?

**Comment by coherit** [comment]
Hmmm....I did not have to set that to True. That's a scary comment too in the app's settings file.

I did set it to True in my settings_local and tried editing the node again and got the same behavior. Works fine on new resources but can't update existing resources on that particular node. 

I tried deleting the new node from the graph and re-adding it. The OVERRIDE_RESOURCE_MODEL_LOCK=True setting allowed me to circumvent the exporting of the resource branch and appending step. However, I got the same behavior upon trying to edit an existing resource.
